client/state/jetpack-connect/reducer.js

```diff
@@ -253,7 +266,7 @@ export function jetpackAuthAttempts( state = {}, action ) {
 			return {};
 		case DESERIALIZE:
 		case SERIALIZE:
-			state;
+			return state;
```

Comment:
Test are good, and I'm for adding tests, but they wouldn't catch this because it's followed by an identical return. These lines are actually redundant. Do we want them to be here for some sort of clarity, or should they be removed?